tests/org.eclipse.fordiac.ide.test.ui/src/org/eclipse/fordiac/ide/test/ui/Abstract4diacUITests.java

```diff
@@ -422,6 +466,195 @@ protected static void deleteFBType(final String typeName) {
 		assertFalse(nodeList.contains(typeName));
 	}
 
+	/**
+	 * Open FB Type in editor from explorer.
+	 *
+	 * @param parentName Name of the parent Project.
+	 * @param typeName   Name of the new type.
+	 */
+	protected static void openFBTypeInEditor(final String parentName, final String typeName) {
+		final SWTBotView systemExplorerView = bot.viewByTitle(SYSTEM_EXPLORER_LABEL);
+		systemExplorerView.show();
+		final Composite systemExplorerComposite = (Composite) systemExplorerView.getWidget();
+		final Tree swtTree = bot.widget(WidgetMatcherFactory.widgetOfType(Tree.class), systemExplorerComposite);
+		final SWTBotTree tree = new SWTBotTree(swtTree);
+		final SWTBotTreeItem parentItem = tree.getTreeItem(parentName);
+		parentItem.expand();
+		SWTBotTreeItem projectItem = null;
+		for (final SWTBotTreeItem item : parentItem.getItems()) {
+			if (item.getText().contains(typeName)) {
+				projectItem = item;
+				break;
+			}
+		}
+		assertTrue(projectItem.getText().contains(typeName));
+	}
+
+	/**
```

Comment:
I think I have a solution for the copied code. Essentially, we need to copy the copyright header if we plan to use the code ourselves. But copying the whole copyright header into this class feels wrong. I therefore suggest to create a new class that has the copied methods and also add the copyright header from SWTBotSiriusHelper